tests/test_dataframe.py

```diff
@@ -258,3 +259,38 @@ def test_dataframe_array_raise(transient_rst):
         ValueError, match="Can only export to array if the DataFrame contains a single"
     ):
         _ = df.array
+
+
+def test_dataframe_min_max():
+    simulation = post.TransientMechanicalSimulation(examples.download_crankshaft())
+    df = simulation.displacement(all_sets=True)
+    # Over the mesh entities
+    min_over_mesh = [[-0.00074732, -0.00040138, -0.00021555]]
+    assert np.all(np.isclose(df.min()._fc[0].data.tolist(), min_over_mesh))
```

Comment:
Is it expected to expose and use a protected attribute `_fc`?

Reply:
@MaxJPRey not to the user. We do have `._core_object` property though which we will implement for all classes wrapping a Core entity to give easy access in case it is needed. 
Here in the tests this allows me to compare the result of the max operation with doing the same thing directly on the data held by the underlying fields container.